updating/preparing-eus-eus-upgrade.adoc

```diff
@@ -14,12 +14,11 @@ There are a number of caveats to consider when attempting an EUS-to-EUS update.
 
 * EUS-to-EUS updates are only offered after updates between all versions involved have been made available in `stable` channels.
 * If you encounter issues during or after upgrading to the odd-numbered minor version but before upgrading to the next even-numbered version, then remediation of those issues may require that non-master hosts complete the update to the odd-numbered version before moving forward.
-* You can do a partial update by updating the worker or custom pool nodes to accommodate the time it takes for maintenance. 
+* You can do a partial update by updating the worker or custom pool nodes to accommodate the time it takes for maintenance.
 * You can complete the update process during multiple maintenance windows by pausing at intermediate steps. However, plan to complete the entire update within 60 days. This is critical to ensure that normal cluster automation processes are completed including those associated with certificate rotation.
 * You must be running at least {product-title} 4.8.14 before starting the EUS-to-EUS update procedure. If you do not meet this minimum requirement, update to a later 4.8.z before attempting the EUS-to-EUS update.
 * Support for RHEL7 workers was removed in {product-title} 4.10 and replaced with RHEL8 workers, therefore EUS-to-EUS updates are not available for clusters with RHEL7 workers.
 * Node components are not updated to {product-title} 4.9. Do not expect all features and bugs fixed in {product-title} 4.9 to be made available until you complete the update to {product-title} 4.10 and enable all MachineConfigPools to update.
-* EUS-to-EUS updates are not applicable to compact three-node clusters and Single Node Openshift (SNO) topologies because all of the hosts in these configurations are control plane hosts.
-
+* All the clusters might update using EUS channels for a conventional update without pools paused, but only clusters with non control-plane `MachineConfigPools` can do EUS-to-EUS update with pools paused.
```

Comment:
Nit: MacineConfigPools should not be in backticks. Is "MachineConfigPools" referring to CRDs in this context? or object? For example: 

* All clusters might update using EUS channels for a conventional update without pools paused, but only clusters with non-control plane `MachineConfigPool` objects can perform EUS-to-EUS updates with pools paused.

either way, it needs a noun after it.

Reply:
Fixed.